modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/itemselector/plugin.js

```diff
@@ -281,13 +284,7 @@
 						: document.getElementById(`cke_${editor.name}`);
 
 					if (typeof callback === 'function') {
-						callback(imageSrc, selectedItem);
-
-						instance._checkImageWidth(
-							editor,
-							editorContent,
-							imageSrc
-						);
+						callback(imageSrc);
```

Comment:
This change was needed for code to work without `adaptivemedia`. It is the same logic, we only want to execute native callback if such callback exists.